javascript/src/main/components/TutorAssignment/TutorAssignmentSelect.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import { Cascader } from "antd";
+import { asHumanQuarter } from "main/utils/quarter";
+
+import {
+  quarterProvider as defaultQuarterProvider,
+  courseProvider as defaultCourseProvider,
+  tutorAssignmentProvider as defaultTutorAssignmentProvider
+} from "main/services/selectorSupport.js"
+
+export const selectorIndices = {
```

Comment:
Is this something we want to export? These constants seem more for internal use.

Reply:
They are needed in the tests.   

Is there another way to reference them in the tests without copying and pasting the code there?

Reply:
In this test:
https://github.com/ucsb-cs156-w21/proj-ucsb-cs-las/blob/9d0d47227e927950161bd1ee31dffb6c57d97a0a/javascript/src/test/components/TutorAssignment/TutorAssignmentSelect.test.js#L40

`onChange` is called with `2` because the id of the selected tutor assignment is `2`. `TUTOR_ASSIGNMENT` (the index where the cascader stores the tutor assignment id) also happens to be `2`, but for a completely different reason.
https://github.com/ucsb-cs156-w21/proj-ucsb-cs-las/blob/9d0d47227e927950161bd1ee31dffb6c57d97a0a/javascript/src/test/components/TutorAssignment/TutorAssignmentSelect.test.js#L20

Reply:
Good Call... I've fixed this by replacing the numbers 1 and 2 in the tests with numbers 17 and 42 that stand out more, and convey to the reader better the relationship between the "arrange" and "assert" parts of the test.

And, I've removed the export.   This is how code review should work :-)